.travis.yml

```diff
@@ -85,16 +85,18 @@ matrix:
         - chmod +x install.sh
         - ./install.sh --user && rm -f install.sh
         - ~/bin/bazel test --test_output=all ${OLC_PATH}:all
+        - wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py && pip install yapf && yapf --diff ${OLC_PATH}/*py
```

Comment:
This command here seems to be failing in Travis CI. Perhaps because it's returning a non-0 status?

Reply:
Down to python formatting. Fixed the format, improved the command so it's a bit more obvious.